src/components/articles/cbd-article.vue

```diff
@@ -0,0 +1,132 @@
+
+<template>
+   <div style="border:none;margin-top:10px">
+        <div v-if="!loading">
+
+            <div v-if="!hideCoverImage && article && article.coverImage && article.coverImage.url">
+                <cbd-article-cover-image cover-image="article.coverImage"></cbd-article-cover-image>
```

Comment:
this component `cbd-article-cover-image` cannot be found

Reply:
fixed